marlowe-playground-client/default.nix

```diff
@@ -2,39 +2,34 @@
 let
   playground-exe = set-git-rev haskell.packages.marlowe-playground-server.components.exes.marlowe-playground-server;
 
-  server-invoker =
-    let
-      # the playground uses ghc at runtime so it needs one packaged up with the dependencies it needs in one place
-      runtimeGhc = haskell.project.ghcWithPackages (ps: [
-        ps.marlowe
-      ]);
-    in
-    pkgs.runCommand "marlowe-server-invoker" { buildInputs = [ pkgs.makeWrapper ]; } ''
-      # We need to provide the ghc interpreter with the location of the ghc lib dir and the package db
-      mkdir -p $out/bin
-      ln -s ${playground-exe}/bin/marlowe-playground-server $out/bin/marlowe-playground
-      wrapProgram $out/bin/marlowe-playground \
-        --set GHC_LIB_DIR "${runtimeGhc}/lib/ghc-${runtimeGhc.version}" \
-        --set GHC_BIN_DIR "${runtimeGhc}/bin" \
-        --set GHC_PACKAGE_PATH "${runtimeGhc}/lib/ghc-${runtimeGhc.version}/package.conf.d" \
-        --set GHC_RTS "-M2G"
-    '';
+  build-playground-exe = "$(nix-build --quiet --no-build-output ../default.nix -A plutus.haskell.packages.marlowe-playground-server.components.exes.marlowe-playground-server)";
+
+  build-ghc-with-marlowe = "$(nix-build --quiet --no-build-output -E '(import ./.. {}).plutus.haskell.project.ghcWithPackages(ps: [ ps.marlowe ])')";
 
+  # Output containing the purescript bridge code
   generated-purescript = pkgs.runCommand "marlowe-playground-purescript" { } ''
     mkdir $out
     ${playground-exe}/bin/marlowe-playground-server psgenerator $out
   '';
 
-  # For dev usage
+  # generate-purescript: script to create purescript bridge code
   generate-purescript = pkgs.writeShellScriptBin "marlowe-playground-generate-purs" ''
     rm -rf ./generated
-    $(nix-build ../default.nix --quiet --no-build-output -A marlowe-playground.server-invoker)/bin/marlowe-playground psgenerator generated
+    $(nix-build --quiet --no-build-output ../default.nix -A plutus.haskell.packages.marlowe-playground-server.components.exes.marlowe-playground-server)/bin/marlowe-playground-server psgenerator generated
```

Comment:
Not sure of my nix, but can the first part of this be replaced with `build-playground-exe`?

Reply:
Oh, yes - absolutely. Thanks for spotting that. I'll fix it right away.